otlp/go.mod

```diff
@@ -1,10 +1,10 @@
 module go.opentelemetry.io/proto/otlp
 
-go 1.14
+go 1.16
```

Comment:
Is this needed?

Reply:
No, only my tooling keeps putting it back :(